packages/fast-development-site-react/src/components/site/category-documentation.tsx

```diff
@@ -28,11 +28,21 @@ class CategoryDocumentation extends React.Component<
     {}
 > {
     public render(): JSX.Element {
-        return (
-            <div className={this.props.managedClasses.documentationPanel}>
-                {this.props.children}
-            </div>
-        );
+        if (this.props.children !== undefined) {
+            return (
+                <div className={this.props.managedClasses.documentationPanel}>
+                    {this.props.children}
+                </div>
+            );
+        } else {
+            return (
+                <div className={this.props.managedClasses.documentationPanel}>
+                    <div style={{ padding: "14px" }}>
+                        <p>No documentation provided.</p>
+                    </div>
```

Comment:
Is there a reason we are using the JSS manager for the wrapper here but not the div? Do we actually need the `<div/>`, or could this just be the paragraph?

Reply:
This is how we are formatting the output of the documentation within the temp folders. If you look at the documentation.tsx you can see the div with style, so this matches that.

Reply:
I think @chrisdholt has a valid point here.  Even though it's how it's been done in other places we should always look to only have minimal required number of elements.  So if `<div/>` is required then great.  But, if it's not then we should eliminate this extra element.

Reply:
Maybe @janechu can shed some light on why the `<div />` was added in this way in the first place? I have no issue changing, just trying to understand the intent.

Reply:
This was done quite a while ago and things have since changed in our dependent packages. I believe we could extrapolate it into its own component. That would require a larger refactor however.

Reply:
Prefer to do this in another [task](https://github.com/Microsoft/fast-dna/issues/1118)

Reply:
Does removing the div break something in this instance? From what I can tell this is *net-new* work and we are just re-using a pre-existing pattern. We have a className, then inline style, and a nested div, but have we tested just removing it to see if we can lighten the load here? It just seems all over the place. 

I'm fine doing a larger refactor for the component stuff; I don't know that it's a priority. But if this PR is a priority we should try and ensure that the code we are checking in is necessary and helpful. Is there a visual difference that we can't overcome by simplifying this and moving the code to the wrapping div?

Reply:
@chrisdholt the `<div></div>` element was there to provide a wrapper for the documentation, but we can use a React.Fragment instead. I've altered the logic to be done in the convert script, as this was working as expected in the dev site, but not on the component sites. Now it just sees if the markdown length is 0 and writes the appropriate message.